requirements_dev.txt

```diff
@@ -9,6 +9,7 @@ pytest==8.3.5
 pytest-html==4.1.1
 pytest-xdist==3.6.1
 pytest-timeout==2.4.0
+pytest-error-for-skips==2.0.2
```

Comment:
This was only use in the CI test, not when running on a computer.  Depending on how a user builds BSK, tests are often skipped.  Can we keep this package requirement only for the CI test builds?

Reply:
Ok, I see the 2nd commit now removes this line.  Please clean up the commit history to not have it in the first commit to begin with?  This branch needs a more streamlined commit history with smaller commit that add a feature at a time.